CLI tools/Update version.py

```diff
@@ -2,12 +2,12 @@
 from pathlib import Path
 
 
-cf_path = Path(Path.home(), "curseforge/minecraft/Instances/Fabulously Optimized/")
-title_screen_path = Path(cf_path, "config/isxander-main-menu-credits.json")
-warning_path = Path(cf_path, "config/fabric_loader_dependencies.json")
+mmc_path = Path.home() / "curseforge/minecraft/Instances/Fabulously Optimized/"
+title_screen_path = mmc_path / "config/isxander-main-menu-credits.json"
+warning_path = mmc_path / "config/fabric_loader_dependencies.json"

```

Comment:
Same as previous comment

Reply:
> Same as previous comment

Same. See my reply to your first comment.